packages/graph/README.md

```diff
@@ -1,6 +1,6 @@
-# The HASH Graph API
+# The HASH Graph Query Layer
 
-To run the HASH Graph API make sure `cargo-make` is installed:
+To run the HASH Graph Query Layer make sure `cargo-make` is installed:
```

Comment:
Fine with this for now. Let's use "Query Layer" consistently for the timebeing, and change from this to something else later if we deem necessary.